docs/smart-contracts/marlowe.md

```diff
@@ -60,15 +60,18 @@ Currently, there is only a single known implementation of Marlowe available for
 
 The well-known versions of the validator script are specified in the [ScriptRegistry.hs](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-runtime/src/Language/Marlowe/Runtime/Core/ScriptRegistry.hs) file.
 
-Version hashes as of June 4, 2023:
+Version hashes as of January 20, 2025:
```

Comment:
```suggestion
Version hashes as of January 20, 2024:
```
Is this a typo?  Or does it indicate the hashes will expire in a year? (and if the latter, I believe it would help to explain this)

Reply:
p.s. committing this change (assuming by deduction a typo) because it refers to https://github.com/input-output-hk/marlowe-cardano/blob/223a5e7e122f0f8fc5ed828f483b797f733cb004/marlowe-runtime/src/Language/Marlowe/Runtime/Core/ScriptRegistry.hs which is a currently dated file & therefore the period of validity cannot be a forward reference.